azure-pipelines.yml

```diff
@@ -47,6 +48,10 @@ jobs:
     condition: and(succeeded(), eq(variables['COMPILER'], 'gcc'))
     artifact: RPCS3 for Linux
 
+  - publish: $(Build.ArtifactStagingDirectory)
+    condition: and(succeeded(), eq(variables['COMPILER'], 'clang'))
+    artifact: RPCS3 for Linux (clang)
+
```

Comment:
Instead of 2 of the same thing, just remove the condition and make the artifact line look like
`artifact: RPCS3 for Linux $(COMPILER)`

Reply:
Maybe you're right but I'm scared that changing its name might interfere with next step (publishing).

Reply:
Ok I'll try, can always revert.